vispy/scene/cameras/base_camera.py

```diff
@@ -224,8 +224,8 @@ def fov(self):
     @fov.setter
     def fov(self, fov):
         fov = float(fov)
-        if fov < 0 or fov >= 180:
-            raise ValueError("fov must be between 0 and 180.")
+        if fov < 0 or fov > 180:
+            raise ValueError("fov must be 0 <= fov <= 180")
```

Comment:
I think this error message is less readable than the previous. Anyways, `between 0 and 180` is colloqualy interpreted as inclusive, or at least I do :P